### PR TITLE
Usman/lp 1356

### PR DESCRIPTION
**Pull request by muak50**

### CODE REVIEW

File: common/djangoapps/philu_commands/management/commands/notify_non_active_users.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    Note: As we know in some cases enrollment process may continues after Course Open Date so if any student
+    enroll to some course after 7 days of course open date, those users will not be notified.
+    """
+
+    def handle(self, *args, **options):
+        courses = get_visible_courses()
+
+        for course in courses:
+
+            today = datetime.now().date()
```
Comment (emraan004) [inline common/djangoapps/philu_commands/management/commands/notify_non_active_users.py]: let's use UTC timezone

File: common/djangoapps/philu_commands/management/commands/notify_non_active_users.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+                    template = MandrillClient.COURSE_ACTIVATION_REMINDER
+                    context = {
+                        'first_name': first_name,
+                        'course_name': course_name,
+                        'course_url': course_url
+                    }
+
+                    MandrillClient().send_mail(template, non_active_user.email, context)
+                    log.info('CELERY-TASK: date_now: %s, course_start_date: %s', today, course_start_date)
```
Comment (emraan004) [inline common/djangoapps/philu_commands/management/commands/notify_non_active_users.py]: Let's correct this log if this is not a celery task

File: common/djangoapps/philu_commands/management/commands/notify_non_active_users.py (hunk 1 above)
Comment (muak50) [reply common/djangoapps/philu_commands/management/commands/notify_non_active_users.py]: date  converted to utc

File: common/djangoapps/philu_commands/management/commands/notify_non_active_users.py (hunk 2 above)
Comment (muak50) [reply common/djangoapps/philu_commands/management/commands/notify_non_active_users.py]: this comment is removed now

**Pull request merged**